Whiskey/en-US/about_Whiskey_CreateFile_Task.help.txt

```diff
@@ -0,0 +1,45 @@
+TOPIC
+    about_Whiskey_CreateFile_Task
+
+
+SUMMARY
+    Creates files.
+
+
+DESCRIPTION
+    The `CreateFile` task creates files. Pass the path to one or more files to create the `Path` property. Pass the content for the file to the `Content` property. If a file already exists, your build will fail. To overwrite any existing file, set the `Force` property to `true`. This task uses PowerShell's `New-Item` cmdlet to create the file and sets its content.  

```

Comment:
Use this documentation:

> The `CreateFile` task creates files. Pass the path to one or more files to create the `Path` property. Empty files will be created at those paths. Each path's parent directory must exist. If you want the task to create each file's parent directories, set the `Force` property to `true`.
> 
> By default, empty files are created. To set the content of a file, set the `Content` property to the content you want in the file. If `Content` is not provided, the file's content is left intact. If `Content` is provided but is empty, the file's contents will be reset to an empty file.
> 
> If you want to update a file's last write time to the current date/time, without changing its content, set the `Touch` property to `true`.

Reply:
I'm wondering if we should call this the `File` task and document its behavior this way:

> The `File` task manages files. Pass the path to one or more files to the `Path` property. If you pass the path to a file that doesn't exist, it will be created, along with all necessary parent directories. The task uses PowerShell's `New-Item` cmdlet to create the file. By default, an empty file is created. To give the file specific content, pass that content to the `Content` property.
>
> If you pass paths of files that exist, or pass a path that contains wildcards, the task won't do anything to those existing files. If you pass the `Touch` property, it will update the file's last write time to the current date/time. If you pass the `Content` property, each file's content will be reset to that value.

I think that's how the task currently behaves, but that above description makes it clear that the task does more than just create files.